### PR TITLE
Docs for keyboard shortcuts

### PR DESCRIPTION
**Pull request by dannysepler**
Figured it might be nice to follow up the shift-selection ticket with a doc page on all the various keyboard shortcuts available to xonsh.

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -440,5 +440,5 @@ def make_events():
 def setup(app):
     from xonsh.pyghooks import XonshConsoleLexer
 
-    app.add_lexer("xonshcon", XonshConsoleLexer())
+    app.add_lexer("xonshcon", XonshConsoleLexer)
```
Comment (dannysepler) [inline docs/conf.py:443]: sphinx said this is supposed to pass a class rather than an instance

Comment (scopatz) [reply docs/conf.py:443]: Yeah, for some reason the classes don't work

Comment (scopatz) [reply docs/conf.py:443]: You might need to leave this line as it was originally

**Comment by scopatz** [comment]
Thanks for putting this in @dannysepler!

**Comment by scopatz** [comment]
Thanks again!

**Pull request merged**